### PR TITLE
fix(ui5-table-cell): allow selection of text nodes

### PR DESCRIPTION
**Pull request by MapTo0**
FIXES: https://github.com/SAP/ui5-webcomponents/issues/3494

### CODE REVIEW

**Review by fifoosid — APPROVED** [review]

**Review by fifoosid — COMMENTED** [review]
Waiting for a review by @SAP/ui5-webcomponents-topic-rl before merging

**Review by niyap — APPROVED** [review]

**Pull request merged**
